packages/storage/test/unit/service.test.ts

```diff
@@ -34,19 +35,21 @@ import {
 import { Location } from '../../src/implementation/location';
 import { newTestConnection, TestingConnection } from './connection';
 import { injectTestConnection } from '../../src/platform/connection';
+import sinonChai from 'sinon-chai';
 
 const fakeAppGs = testShared.makeFakeApp('gs://mybucket');
 const fakeAppGsEndingSlash = testShared.makeFakeApp('gs://mybucket/');
 const fakeAppInvalidGs = testShared.makeFakeApp('gs://mybucket/hello');
 const testLocation = new Location('bucket', 'object');
+use(sinonChai);
 
 function makeGsUrl(child: string = ''): string {
   return 'gs://' + testShared.bucket + '/' + child;
 }
 
 describe('Firebase Storage > Service', () => {
-  before(() => injectTestConnection(newTestConnection));
-  after(() => injectTestConnection(null));
+  // before(() => injectTestConnection(newTestConnection));
```

Comment:
did you mean to leave this commented out?

Reply:
Shoot, meant to bring it back